tools/serve/serve.py

```diff
@@ -663,7 +663,17 @@ def load_config(override_path=None, **kwargs):
                u"www1",
                u"www2",
                u"天気の良い日",
-               u"élève"}
+               u"élève",
+               u"deeper",
```

Comment:
OK, o I"m going to sound like the most boring person, but I don't really like the whimsical names.

Is there something wrong with using the existing hosts and adding e.g. 
```
sub1a.www
sub1b.www
sub2a.sub1a.www
sub2b.sub1a.www
sub3a.sub2a.sub1a.www
```
and so on for whatever combinations you need?

Reply:
You're no fun!

Yours insincerely,

gsnedders.